_dashboards/maps.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Maps
+nav_order: 15
+---
+
+# Maps
+
+With OpenSearch Dashboards, you can create maps to visualize your geographical data. OpenSearch lets you construct map visualizations with multiple layers, combining data across different indexes. You can build each layer from a different data source either directly or using simple or ML-powered aggregations. Additionally, you can configure maps to show specific data at different zoom levels. OpenSearch maps are powered by the OpenSearch maps service, which uses vector tiles to render maps. 
+
+## Getting started
+
+To get started, perform the following steps:
+
+1. On the top menu bar, go to **OpenSearch Dashboards > Maps**.
+1. Select the **Create map** button. You can now see the default OpenSearch basemap.
+1. To examine the **Default map** layer configuration, in the **Layers** panel on the upper left of the map, select **Default map**.
+![Default map]({{site.url}}{{site.baseurl}}/images/maps/maps-default.png)
+
+To hide the **Layers** panel, select the collapse (arrow) icon in the panel's upper-right corner.
+{: .tip}
+
+## Layer settings
+
+To change the default map settings, select **Default map** in the **Layers** panel. Under **Layer settings**, you can change the layer name and description and configure zoom levels and opacity for your layer:
+
+- **Zoom levels**: If you want to make a layer visible only on a certain range of zoom levels, you can specify the zoom levels for this layer either by entering them in the text boxes or by sliding the range slider to the desired values.
+
+- **Opacity**: If your map contains multiple layers, one layer can obscure another one. In this case, you may want to reduce the opacity of the top layer so you can see both layers at the same time.
+
+## Adding layers
+
+To add a layer to the map, in the **Layers** panel, select the **Add layer** button.
+
+<img src="{{site.url}}{{site.baseurl}}/images/maps/add-layer.png" alt="Add layer">
+
+You can add either **data layers** or **reference layers** to the map:
+- **Data layers** let you visualize data from various data sources. 
+
+- **Reference layers** serve as basemaps. To use your own or a third-party map as a basemap, [add it as a **Custom map**](#adding-a-custom-map).
+
+## Adding a custom map
+
+OpenSearch supports Web Map Service (WMS) or Tile Map Service (TMS) custom maps. To add a TMS custom map, perform the following steps:
+
+1. In the **Layers** panel, select the **Add layer** button.
+1. From the **Add layer** dialog, select **Reference layer > Custom map**.
+
+    <img src="{{site.url}}{{site.baseurl}}/images/maps/custom-map.png" alt="Add custom map">
+
+1. In the **Custom type** dropdown, select **Tile Map Service (TMS)**. 
+1. Enter the TMS URL.
+1. (Optional) In **TMS attribution**, enter a TMS attribution for the basemap. For example, if you're using Maptiler, enter `Maptiler`. The name `Maptiler` will be displayed in the lower-right corner of the map.
```

Comment:
We don't have to give Maptiler as an example, we can use something else "custom-map"